js/bookmarks.js

```diff
@@ -0,0 +1,9 @@
+"use strict";
+console.log("hello newbie, bookmarks");
+
+// get from local storage or empty array
+const cards = JSON.parse(localStorage.getItem("cards")) ?? [];
+// call function from renderCards.js
+
+const filteredCards = cards.filter((card) => card.bookmarked === true);
```

Comment:
could be even shorter. Up for you to decide if it makes it easier or harder to read ;) 

```suggestion
const filteredCards = cards.filter((card) => card.bookmarked);
```